tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -860,6 +862,7 @@ class GaiaTestCase(MarionetteTestCase, B2GTestCaseMixin):
     def __init__(self, *args, **kwargs):
         self.restart = kwargs.pop('restart', False)
         self.locale = kwargs.pop('locale')
+        self.capture = kwargs.pop('capture', False)
```

Comment:
You mean that line isn't needed then? Then I guess the self.restart and self.locale lines are not needed, either?